<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindPassword</class>
 <widget class="QWidget" name="FindPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QPushButton" name="pushButton_modify">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>更改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>输入确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_return">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>请输入手机号码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_tel">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>请输入emil：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_emil">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>请输入新密码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
        <property name="text">
         <string>请确认：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_passwordsure">
        <property name="styleSheet">
         <string notr="true">color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
